server/src/core/search/search_executor/postprocess.rs

```diff
@@ -232,3 +229,18 @@ fn parse_room_formats(
         None
     }
 }
+
+fn generate_subtext(
+    hit: &meilisearch::MSHit,
+) -> String {
```

Comment:
I have not tested, if rustfmt likes this, but I assume it does
```suggestion
fn generate_subtext(hit: &meilisearch::MSHit) -> String {
```